proposals/models.py

```diff
@@ -231,7 +231,7 @@ class Proposal(models.Model):
     )
 
     pre_approval_institute = models.CharField(
-        _('Welk instituut heeft de studie goedgekeurd?'),
+        _('Welk instituut heeft de aanvraag goedgekeurd?'),
```

Comment:
Dit voelt raar. Een aanvraag is in mijn ogen een FETC aanvraag; dus dit voelt alsof een ander instituut een FETC aanvraag heeft beoordeeld?

'Van welk instituut heb je al goedkeuring gekregen?' voelt dan beter